templates/tests/test-templates.sh

```diff
@@ -98,8 +102,8 @@ function deployTemplate {
         azd env new "$3" --subscription "$4" --location "$5" --no-prompt
     fi
 
-    echo "Create provision preview for $3..."
-    azd provision -e "$3" --preview
+    # echo "Create provision preview for $3..."
+    # azd provision -e "$3" --preview
```

Comment:
I think we still want to keep this?

Reply:
There is a bug i logged regarding this.  
https://github.com/Azure/azure-dev/issues/3447

However in the template tests does preview provide any real value outside of validating the command works? If that is the intent then we can leave it in but we should address the bug mentioned above.